circuit/js/src/cliHandler/compile.ts

```diff
@@ -34,6 +35,8 @@ export const compile = async (
         shouldTime: options.stats,
         inputSchema: f.inputSchema,
         results: cache,
+        capacity: f.config?.capacity,
+        config: f.config?.circuit,
```

Comment:
do we need to pass capacity and config into the `prove` cliHandler as well?

Reply:
also is `config: f.config?.circuit` correct here?

Reply:
discussed w/ Roshan offline; we can rename the `circuit` field in `AxiomV2CircuitConfig` to `config` to keep w/ convention